src/commands/note.ts

```diff
@@ -33,7 +33,7 @@ const noteCommand: Command = {
   isActive: state => {
     const { cursor } = state
     const path = cursor ? simplifyPath(state, cursor) : HOME_PATH
-    return attribute(state, head(path), '=note') !== null
+    return hasNotePath(state, head(path))
```

Comment:
The note command should be active when the thought has an actual note. Just having a note path is not enough. So it should be:

```ts
return noteValue(state, head(path)) !== null
```